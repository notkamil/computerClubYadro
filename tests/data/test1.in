1
09:00 20:00
10
09:15 1 client1
09:30 2 client1 1
18:00 4 client1