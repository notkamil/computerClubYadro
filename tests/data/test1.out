09:00
09:15 1 client1
09:30 2 client1 1
18:00 4 client1
20:00
1 90 08:30